### PR TITLE
Backend Cleanup

### PR DESCRIPTION
**Pull request by ColbyFoster3**
- I cleaned up the fetch and post calls for all models and quickly tested them to make sure they all work
- I cleaned up the db to ensure there is no bad data that is missing keys

### CODE REVIEW

File: server/routes/business.js [hunk 1]
```diff
@@ -54,10 +54,12 @@ router.post('/create', async (req, res) => {
     //Check if owner already has a business
     let ownerHasBusiness = await Business.findOne({ownerId: req.body.ownerId}).exec();
```
Comment (bflemin1) [inline server/routes/business.js]: We should avoid this: Prefer using top-down structure, searching for the business' flag in an Owner entry.

**Review by dunc415 — APPROVED** [review]

File: server/routes/employee.js [hunk 2]
```diff
@@ -3,40 +3,69 @@ const router = express.Router();
...
 });
 
 /*
-    Posts an employee with the email and isManager
+    * DONE
+    Posts an employee with an email and isManager
 */
 router.post('/create', async (req, res) => {
+    //check if req body exists 
     if(!req.body) return res.status(400).send({err: 'No request body'});
```
Comment (bflemin1) [inline server/routes/employee.js:41]: Just a general comment, I think we do this everywhere. This only checks that there is indeed a body. If they give a bad body in the request some error will fire that we may not be checking for or worse we might make an incorrect db entry.

File: server/routes/items.js [hunk 3]
```diff
@@ -38,14 +38,16 @@ router.post('/get', function(req, res){
                     return res.status(201).send({formattedItem, code: 201});
                 }
             });
+        } else {
+            return res.status(400).send({err: 'Type 2: Id is not a valid ObjectId', code: 403});
```
Comment (bflemin1) [inline server/routes/items.js:42]: This probably makes sense, but I might need an explanation, why are there two error types?

**Review by bflemin1 — COMMENTED** [review]
Looks great just a couple comments.

**Pull request merged**